packages/backend/src/apps/formsg/triggers/new-submission/get-data-out-metadata.ts

```diff
@@ -110,8 +113,9 @@ function buildAnswerArrayMetadatum(
     case 'table':
       return buildAnswerArrayForTable(fieldData)
     default:
-      logger.warn('Unknown fieldtype in answer array', {
-        fieldData,
+      logger.warn(`Answer array unknown fieldtype: ${fieldData.fieldType}`, {
```

Comment:
nit: we can put in fieldType inside the object as well :)

Reply:
yessir